Project-Aurora/Project-Aurora/Utils/PointerUpdateUtils.cs

```diff
@@ -0,0 +1,121 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Diagnostics;
+using System.Linq;
+using System.Text;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Octokit;
+
+namespace Aurora.Utils
+{
+    /// <summary>
+    /// Class to update pointers for games with memory reading functionality.
+    /// </summary>
+    public static class PointerUpdateUtils
+    {
+        private static GitHubClient gClient = new GitHubClient(new ProductHeaderValue("aurora-pointer-updater"));
+
+        /// <summary>
+        /// Updates the Pointers directory with the most recent from a specified branch.
+        /// </summary>
+        /// <param name="branch">The branch in antonpup/Aurora to pull the pointers from.</param>
+        /// <param name="useOctokit">Get the pointers through GitHub's Content API using Octokit. Recommend this to be false to avoid rate limiting by GitHub's API.</param>
+        /// <param name="backgroundTask">Set updating pointer files as a background task.</param>
+        /// <returns></returns>
+        public static void UpdatePointers(string branch, bool useOctokit = false, bool backgroundTask = false)
+        {
+            if (!backgroundTask) Task.Run(() => FetchAllPointers(branch, useOctokit)).Wait();
+            else Task.Run(() => FetchAllPointers(branch, useOctokit));
+        }
+
+        public static void UpdateLocalPointers(string branch, bool backgroundTask = false)
+        {
+            if (!backgroundTask) Task.Run(() => FetchLocalPointers(branch)).Wait();
+            else Task.Run(() => FetchLocalPointers(branch));
+        }
+
+        private static async Task FetchAllPointers(string branch, bool useOctokit)
+        {
+            // Update pointer files in Aurora/Pointers/
+            string pointerPath = Path.GetDirectoryName(Process.GetCurrentProcess().MainModule.FileName);
+            pointerPath += @"\Pointers\";
```

Comment:
Minor suggestion: Use `string pointerPath = Path.Combine(Global.ExecutingDirectory, "Pointers");` This is what the pointer readers use to resolve their path.

Reply:
Thanks!